docs/manuals/core/versioning/production_mode.md

```diff
@@ -18,76 +18,217 @@ To convert an experiment version to a production, you can run the Taipy applicat
+attached to it. When listing the existing versions, we can see that 1.0 is the latest version used.
 Therefore, when running the Taipy application in production mode without providing the version name,
 the latest is used and converted to production before running.
 
 Note that once the version is converted to production, the application can access all production
 entities, including from older production versions.
 
-
 # Remove a production version
 
 To remove a production version, you can use the `taipy manage-versions` with the `--delete-production`
 option on the Taipy CLI and providing the version name (see the
 [Manage versions on Taipy CLI page](../../cli/manage-versions.md) for more details).
 
 ```console
-$ taipy manage-versions --delete-production 2.0
-Successfully delete version 2.0 from production version list.
+$ taipy manage-versions --delete-production 0.1
+Successfully delete version 0.1 from production version list.
 
 $ taipy manage-versions --list
 Version number                         Mode                   Creation date
-2.0                                    Experiment (latest)    2023-01-25 13:06:00
-1.0                                    Production             2023-01-25 13:00:05
+1.0                                    Production (latest)    2023-01-25 13:06:00
+0.1                                    Experiment             2023-01-25 13:00:05
 325d0618-6f9e-459b-9597-48fa93a57a23   Experiment             2023-01-25 12:20:56
 26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
 ```
 
-After running the commands above, version 2.0 is an experiment version again. It is no longer a
+After running the commands above, version 0.1 is an experiment version again. It is no longer a
 production version.
+
+# Change config of an existing production version
+
+Similar to experiment mode, to re-run an existing production version, the configuration
+of the application must not have any breaking conflict.
+
+Let's assume multiple changes have been made to the configuration in `main.py`.
+
+```python linenums="1" title="main.py"
+{%
+include-markdown "./code_example/main_with_multiple_changes.py"
+comments=false
+%}
+```
+
+```console
+$ python main.py --production 1.0
+[2023-01-25 12:52:05,484][Taipy][ERROR] The version 1.0 Configuration is conflicted with the current Configuration:
+    DATA_NODE "input" has attribute "path" added: input.pkl
+    DATA_NODE "output" has attribute "path" added: output.pkl
+    DATA_NODE "input" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    DATA_NODE "output" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    TASK "example_algorithm" has attribute "skippable" modified: False:bool -> True:bool
+
+Please add a new production version with migration functions or run your application with --taipy-force option to override the Config of production version 1.0.
```

Comment:
"override the Config of production" -> "override the production configuration"
That would be code.
And you've been using "configuration" all the time so far, not "Config".

Reply:
True. Let's update it here then I will update the code latter.

```suggestion
Please add a new production version with migration functions or run your application with --taipy-force option to override the production Configuration of version 1.0.
```